app/src/main/java/org/mozilla/fenix/components/toolbar/BrowserToolbarController.kt

```diff
@@ -81,6 +81,9 @@ class DefaultBrowserToolbarController(
     private val onCloseTab: (Session) -> Unit
 ) : BrowserToolbarController {
 
+    private val useNewSearchExperience
+        get() = activity.settings().useNewSearchExperience
```

Comment:
We're moving to injecting dependencies in the constructor instead, `Settings` should be passed in via the constructor.

Reply:
Let's put this into the next piece of the work @boek